### DISCUSSION TITLE
Document fields

### DISCUSSION THREAD

**Discussion by agloolik** [keystonejs/keystone#6596] [Event Q&A Q3 2021]
Will the document-field support adding images and videos ? I wish they would render in the admin page rather than a select box.

**Comment by raveling** [comment]
Thanks for your question @agloolik 🙂 
We answered this in the recent community Q&A:
https://www.youtube.com/watch?v=r1IJh-iHm1c&t=2340s

**Reply by pc-erin** [reply]
"This video isn't available anymore"
